portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,41 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+import java.util.ArrayList;
```

Comment:
please order java imports according to these rules: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Specifically, this section should be in alphabetical order.

Reply:
this still needs to be sorted with the lines above

Reply:
still not quite right - this needs to go on line 18.